Source/WebKit/Shared/UserData.h

```diff
@@ -47,14 +47,14 @@ class UserData {
         virtual bool shouldTransformObject(const API::Object&) const = 0;
         virtual RefPtr<API::Object> transformObject(API::Object&) const = 0;
     };
-    static RefPtr<API::Object> transform(API::Object*, const Transformer&);
+    static RefPtr<API::Object> transform(const RefPtr<API::Object>&, const Transformer&);
```

Comment:
I am curious how @rniwa feels about this. Last I heard, we were not making changes to parameters types like this.

Reply:
There are some precedents, for example, see: `BytecodeGenerator::BytecodeGenerator` or `VariableEnvironment::markVariableAsCapturedIfDefined` etc.

Reply:
I don't think we should do this.

Reply:
I'm happy to change this, but can you expand on the reasoning behind that?